modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -75,6 +78,10 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BrowserWebDriverContainer.class);
 
+    /**
+     * @deprecated please use {@link BrowserWebDriverContainer#BrowserWebDriverContainer(DockerImageName)}
+     */
+    @Deprecated
```

Comment:
note that, if we deprecate this, there are no constructors that would *not* set `recordingMode = SKIP` by default.

Reply:
Isn't this even in line with our current docs?
https://www.testcontainers.org/modules/webdriver_containers/#recording-videos
> By default, no videos will be recorded. However, you can instruct Testcontainers to capture videos for all tests or just for failing tests.